docs/getting-started.md

```diff
@@ -1,44 +1,46 @@
-* [Getting Started with TypeScript](#getting-started-with-typescript)
-* [TypeScript Version](#typescript-version)
+* [TypeScript ile Başlamak](#typescripte-baslarken)
```

Comment:
TypeScript ile Başlamak => TypeScript'e Başlarken

Bağlantı verilen başlık ile aynı olması için.